Reorg Polkadot Vision (#6729)
w3f/polkadot-wiki: docs/general/polkadot-v1.md

[Polkadot runtime v1.0.0](https://github.com/paritytech/polkadot/releases/tag/v1.0.0). This section
focuses on Polkadot 1.0 and some philosophical digressions about network resilience and blockspace.

> Polkadot is a blockchain-based decentralized computer that brings to the multi-chain vision the following innovations:

Reviewer: Love this!

- **Rollup Chains (parachains).** These are application-specific chains connected to Polkadot. Polkadot is a sharded network
  where transactions are processed in parallel within each shard. Continuing the comparison of Polkadot being a decentralized computer, Polkadot has a virtual CPU with virtual cores (i.e., shards). Polkadot cores are the gateway for applications such as rollup chains to use Polkadot as a service.

>     !!!info "Rollup chains where named parachains"

Reviewer: ```suggestion
    !!!info "Rollups or Parachains?"
```

>         Polkadot rollup chains were named parachains. This reflected the technical nature of parachains, which are blockchains running in parallel on Polkadot shards. Due to the broad industry acceptance of the term "rollup", parachains are, in a way, rollup chains: fully-fledged blockchains sending data to Polkadot for security guarantees. A comparison between Polkadot rollups and other rollups is available [here](../learn/learn-comparisons-rollups.md#rollup-comparison-table).

Reviewer: Maybe we should link ELVES here too?

```suggestion
        Polkadot rollups used to be referred to as parachains in the past. This reflected the technical nature of parachains, which are blockchains running in parallel as Polkadot shards. However, the technological nature of parachains are akin to a rollup protocol. Due to the broad industry acceptance of the term "rollup", parachains are, in a way, rollup chains: fully-fledged blockchains utilizing Polkadot's rollup mechanism for security guarantees. A comparison between Polkadot rollups and other rollups is available [here](../learn/learn-comparisons-rollups.md#rollup-comparison-table).
```

  
- **Shared security**. Any rollup chain attached to a Polkadot
  virtual core can benefit from Polkadot's shared security and interoperability model. This means the Polkadot
  [Nominated-Proof-of-Stake (NPoS)](../learn/learn-staking.md#nominated-proof-of-stake-npos)
  mechanism along with its [protocol ELVES](../learn/learn-parachains-protocol.md), can secure fully-fledged blockchains out-of-the-box without having to bootstrap security on their own.

> - **Secure interoperability.** Any rollup chain chain attached to a Polkadot core (as well as Layer-two chains built on top

Reviewer: Are there any examples of layer-two chains that utilize a parachain?

Author: for example a dApp deployed as a smart contract on Astar or Moonbeam

  of them) can benefit from Polkadot's native interoperability via [XCM](../learn/learn-xcm.md) and can thus communicate and exchange value and information with other Polkadot rollup chains.
- **Resilient infrastructure.** This is achieved by keeping the network decentralized without
  compromising scalability and throughput, and through the on-chain
  [treasury](../learn/learn-polkadot-opengov-treasury.md) funds that can be accessed through
  [OpenGov](../learn/learn-polkadot-opengov.md) referendum. Those funds guarantee constant sponsorship for network upgrades, educational material, software development, events, and initiatives.
- **Fast blockchain development.** This is achieved through the modular and

>   flexible [Polkadot SDK Substrate](./glossary.md#substrate).  Builders can create fully-fledged blockchains designed explicitly around their application and value proposition, while Polkadot takes care of security and interoperability.

Reviewer: ```suggestion
  flexible [Substrate, located within the Polkadot SDK](./glossary.md#substrate).  Builders can create fully-fledged blockchains designed explicitly around their application and value proposition, while Polkadot takes care of security and interoperability.
```

- **Fostering next-gen of Web3 core developers.** This is achieved through different initiatives
  such as [the Polkadot Blockchain Academy](https://polkadot.network/development/academy/) and [Polkadot Alpha Program](./alpha-program.md).

### Polkadot's Representation

> Polkadot has a relay chain acting as the main chain of the system. The Polkadot relay chain is

Reviewer: I would remove this line and just jump straight into it, IMO
```suggestion
```

Polkadot's rollup chains construct and propose blocks to validators on the relay chain, where the blocks undergo
rigorous [availability and validity](../learn/learn-parachains-protocol.md) checks before being
added to the finalized chain. As the relay chain provides the security guarantees,
[collators](../learn/learn-collator.md), full nodes of these rollup chains, do not have any security
responsibilities, and thus do not require a robust incentive system. This is how the entire network
stays up to date with the many transactions.

The [Cross-Consensus Messaging Format (XCM)](../learn/learn-xcm.md) allows rollup chains to send
messages of any type to each other. The relay chain's shared security and validation logic 
provides the environment for trust-free message passing that opens up trustless interoperability.

To interact with chains that want to use their finalization process (e.g., Bitcoin),
Polkadot has [trustless bridges](../learn/learn-bridges.md) that offer two-way compatibility, meaning that transactions can happen between Polkadot and other ecosystems without relying on centralized parties.

### Polkadot's Additional Functionalities

> The Polkadot relay chain managed parachains' crowdloans and auctions. Relay chain virtual cores were leased via auctions in 3-month

Reviewer: Perhaps lets add a note to say this has been / is being replaced by coretime

chunks for a maximum of two years, and crowdloans allowed users to loan funds to teams
for lease deposits in exchange for pre-sale tokens. There was no other way you could access cores on
Polkadot 1.0.

> The relay chain also managed other tasks such as [staking](../learn/learn-staking.md),

Reviewer: I guess we are using the past tense here to indicate that it won't be around much longer? (i.e., managed by the Polkadot Hub instead)

Author: This is about Polkadot 1.0; the whole page is written in the past tense

Ideally, a network should be decentralized "enough" to make it practically impossible for someone to
exert manipulative or malicious influence on the network. So, **decentralization is a tool while the

> goal is resilience**, which is achieved by additionally providing on-chain treasury and open governance

Reviewer: ```suggestion
goal is resilience**, which is achieved by additionally providing an on-chain treasury and open governance
```

mechanism allowing continuous incentives for the network's participants without relying on
intermediaries or centralized entities.

Polkadot 1.0 achieved resilience through the following strategies:

- **Nominated Proof-of-Stake (NPoS)** where the stake per validator is maximized and evenly
  distributed across validators, and participation in native staking is decentralized and democratized as much as possible.

> - [**ELVES protocol**](../learn/learn-parachains-protocol.md) allows for the shared security system and the "pessimistic" rollup system, where everything coming through Polkadot's virtual cores passes rigorous checking.

Reviewer: Tiny nit, but maybe we can call it cynical, as per this ;) https://x.com/baderyo_o/status/1937901346780479548

Author: I am a bit reluctant to use the term "cynical" that hasn't been widely adopted yet. Imo it is better to stick with Polkadot rollup for now

blockchain's security, computing, and storage capabilities as an end product. Blockspace produced by
different blockchains can vary in security, flexibility, and availability.

- **Security** is intended to show how secure blockspace is. In Proof-of-Stake (PoS) networks, this is
  directly related to how much stake is locked on validator nodes, how much variance in stake there
  is between validators (i.e., how easy it is to attack a single validator), and how many validators
  there are securing the network (i.e., how easy it is for colluding validators to exert influence on
  the network). Additionally, it is also important to look at how many validators are owned by
  single operators (this will determine the degree of social centralization of the network), and how
  many validators run on the same service provider (this will determine the degree of physical
  centralization of the network).
- **Flexibility**, intended as how flexible the blockspace is, what can be done with it, and what
  type of data can be stored. Data quality plays an important role depending on the type of network.

>   One might avoid having situations in which poor quality data floods blockspace, hindering the prompt

Reviewer: This probably is moreso in the realm of security, as security / anti-spam ensures that legit data / blockspace is propgated
